### PR TITLE
task-9352: backup notice screen - handle event

### PR DESCRIPTION
**Pull request by vanpd-0859**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9352
### Goals :soccer:

- Click Backup Later to go to Wallet screen
- Click Backup Mnenomic Phrase to go to Backup New Wallet Screen

### CODE REVIEW

**Comment by vanpd-0859** [comment]
ready

File: CryptoWallet/Scenes/Login/CreateWallet/BackupNotice/BackupNoticeViewController.swift [hunk 1]
```diff
@@ -10,13 +10,18 @@ import UIKit
 import Reusable
 
 final class BackupNoticeViewController: UIViewController {
-    var wallet: Wallet?
+    var mnonemics: String = ""
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Login/CreateWallet/BackupNotice/BackupNoticeViewController.swift]:
```suggestion
    var mnonemics = ""
```

Comment (tanbtd-1747) [inline CryptoWallet/Scenes/Login/CreateWallet/BackupNotice/BackupNoticeViewController.swift]:
```suggestion
    var mnemonic = ""
```

**Comment by vanpd-0859** [comment]
ready

**Review by tanbtd-1747 — APPROVED** [review]

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**